src/home_robot/home_robot/utils/simple_env.py

```diff
@@ -5,25 +5,87 @@
 from abc import ABC, abstractmethod
 from typing import Any, Dict, Optional
 
+import matplotlib.pyplot as plt
 import numpy as np
 
 from home_robot.core.abstract_env import Env
 from home_robot.core.interfaces import Action, Observations
+from home_robot.motion.space import ConfigurationSpace
 
 
 class SimpleEnv(Env):
     """Simple 2D environment for testing."""
 
-    def __init__(self, seed: int = 0):
-        self.reset(seed)
+    def __init__(self, obs: np.ndarray = None, size=10, obstacle_size=6, seed: int = 0):
+        """Create simple 2d env. obs is obstacle location."""
+        self.size = size
+        self.obstacle_size = obstacle_size
+        if obs is None:
+            self.reset(seed)
+        else:
+            self.obstacle_pos = obs
+
+    def get_space(self) -> ConfigurationSpace:
+        """Get a space that we can use for planning"""
+        return ConfigurationSpace(2, np.zeros(2), np.ones(2) * self.size, step_size=0.5)
+
+    def validate(self, q):
```

Comment:
If this is going to be reused might be helpful to have typehint + shapehint for q

Reply:
added  but this is just for unit tests basically